less/index.less

```
@@ -52,8 +52,155 @@ table {
 * {
     box-sizing: border-box;
 }
+body {
+    width:70%;
+    margin: 0 auto;}
+
+html {font-size: 63%;}
 
 html, body {
     height: 100%;
 }
 
+@small-font: 1.6rem;
+@medium-font: 2rem;
+@medium-large-font: 3.5rem;
+@large-font: 6rem;
+
```

```suggestion
@small-font: 1.6rem;
@medium-font: 2rem;
@medium-large-font: 3.5rem;
@large-font: 6rem;
```
awesome job on using variables